src/app/App.tsx

```diff
@@ -102,7 +104,15 @@ function DropZoneArea() {
     <RootLayout>
       <DropZoneContainer
         onDrop={(files) => {
-          void loadFiles(files, dispatch);
+          if (
+            isEqual(appState, getEmptyAppState()) &&
```

Comment:
I think you can remove this check. We want to replace the state entirely in that case (later we will surely implement a confirmation since it's destructive)